File: elemi_programozasi_tetelek/osszegzes/osszegzes.cs
using System;
using System.Collections.Generic;

namespace ConsoleApp1
{
    internal class Program
    {
        static double Osszeg1(List<double> lista)
        {
            double sum = 0;
            foreach (double elem in lista)
            {
                sum += elem;
            }
            return sum;
        }
        static double Osszeg2(List<double> lista)
        {
            double sum = 0;
            for (int i = 0; i < lista.Count; i++)
            {
                sum += lista[i];
            }
            return sum;
        }

        static void Main(string[] args)
        {
            List<double> l = new List<double> { 1, 2, 3, 4, 5};
            Console.WriteLine(Osszeg1(l));
            Console.WriteLine(Osszeg2(l));
        }
    }
}

File: elemi_programozasi_tetelek/osszegzes/osszegzes.js
function Osszeg1(lista){
    sum = 0;
    for (const elem of lista) {
        sum += elem;
    }
    return sum;
}

function Osszeg2(lista){
    sum = 0;
    for (let i = 0; i < lista.length; i++) {
        sum += lista[i];
    }
    return sum;
}

l = [1, 2, 3, 4, 5];
console.log(Osszeg1(l));
console.log(Osszeg2(l));
